MAUI/TreeMap/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of .NET MAUI TreeMap | Syncfusion
+description: Learn about the introduction of Syncfusion .NET MAUI TreeMap (SfTreeMap) in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Overview of .NET MAUI TreeMap (SfTreeMap)
+
+The Syncfusion .NET MAUI TreeMap control allows you to visually represent hierarchical data with rectangles that are sized and colored based on underlying values.
+
+## Key features 
+ 
+ * Seamlessly bind the TreeMap control to hierarchical data structures.
+ * Group the TreeMap levels based on the data hierarchy.
+ * Offers multiple layout options, including `Squarified`, `SliceAndDiceAuto`, `SliceAndDiceHorizontal`, and `SliceAndDiceVertical`, to represent items at different levels of a hierarchy.
+ * Supports interactive highlighting, selection, and tooltips.
+ * Includes various legend styles to provide information about the TreeMap data clearly.
+ * Completely customize the TreeMap leaf items with templates.
```

Comment:
Use this sentence.

* Seamlessly bind the TreeMap control to hierarchical data structures.
 * Offers various layout options such as Squarified, SliceAndDiceHorizontal, SliceAndDiceVertical, and SliceAndDiceAuto.
 * Offers multi-level support for hierarchical collection data.
 * Utilize brush settings within TreeMap to customize fill colors for leaf items based on ranges or values.
 * Enhance TreeMap interpretation with the legend feature, incorporating various colors, shapes, and more.
 * Interactive tooltip support to provide additional details about the tree map items.
 * Allows you select or highlight the specific tree map items.
 * Provided accessibility support for tree map control.
 * Implemented keyboard support to perform navigation between items of tree map control.
 * Customize tree map appearances.

Reply:
Content changed